### DISCUSSION TITLE
Blade component should be created with proper class name

### DISCUSSION THREAD

**Discussion by xorock** [laravel/framework#47979] [Ideas]
In addition to the basic command 
`php artisan make:component Forms/Input` it is possible to generate components by quick form
`php artisan make:component forms.input`. Unfortunately, it generates a class named `forms.input extends Component`. 
Maybe it would be good to change it to the correct camelCase, namespaced form?

**Comment by TheBlckbird** [comment]
The problem lies in the GeneratorCommand class, it's nothing specific to the `make:component` command.
I don't know if it is wanted that you can create every class (like controllers) with this dot notation.

**Comment by TheBlckbird** [comment]
See #48014
